lib/rubocop/cop/rails/dot_separated_keys.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop enforces the use of dot-separated locale keys instead of specifying the `:scope` option
```

Comment:
Can you tweak it? Probably the latest test will fail. https://github.com/rubocop/rubocop-rails/commit/914e35f3a24f26b4448e19752eaabb8567d43d53

```suggestion
      # Enforces the use of dot-separated locale keys instead of specifying the `:scope` option
```